pkg/apis/config/v1alpha1/types.go

```diff
@@ -20,7 +21,8 @@ type Configuration struct {
 	// +optional
 	HealthCheckConfig *healthcheckconfigv1alpha1.HealthCheckConfig `json:"healthCheckConfig,omitempty"`
 	// CosignPublicKeys is the cosign public keys used to verify image signatures.
-	CosignPublicKeys []string `json:"cosignPublicKeys,omitempty"`
+	// +optional
+	CosignPublicKeys *runtime.RawExtension `json:"cosignPublicKeys,omitempty"`
```

Comment:
I'm unsure why this can't be typed to a specific struct?

Reply:
Lakom extension controller config should not be coupled to the lakom admission controller config, a good practice is to embed such configurations as `runtime.RawExtension`.